app/javascript/driving/components/ActiveRide.js

```diff
@@ -128,11 +131,17 @@ const ActiveRide = props => {
         <p>Total Passengers: {passengers}</p>
         <p>Special requests: {special_requests}</p>
       </div>
-      <div className="bottom-controls secondary">
-        {renderButtons(props)}
-      </div>
+      <div className="bottom-controls secondary">{renderButtons(props)}</div>
     </div>
   );
 };
 
+ActiveRide.propTypes = {
+  ride: RidePropTypes,
```

Comment:
I can't think of a situation where we'd want to render this component without a ride object. There _is_ an `isRequired` on this but it's obfuscated because of the `RidePropTypes` import.